### PR TITLE
Switch experimental to rds iam

### PR DESCRIPTION
**Pull request by chrisgilmerproj**
## Description

This flips the switch to using RDS IAM for DB authentication in experimental. 

## Setup

The success criteria here is that the application can successfully connect to do a migration. Then we'll upload some fake orders to the DB which should verify that the app can take and store data. Finally we'll check using the `orders-api-client` that the orders are indeed there.

## Code Review Verification Steps

* [x] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [x] Request review from a member of a different team.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -34,7 +34,18 @@ dev_up: ## Start development environment
...
+		-e AWS_ACCESS_KEY_ID \
+		-e AWS_ACCOUNT_ID \
+		-e AWS_DEFAULT_REGION \
+		-e AWS_REGION \
+		-e AWS_SDK_LOAD_CONFIG \
+		-e AWS_SECRET_ACCESS_KEY \
+		-e AWS_SECURITY_TOKEN \
+		-e AWS_SESSION_EXPIRATION \
+		-e AWS_SESSION_TOKEN \
+		dev /bin/bash
```
Comment (chrisgilmerproj) [inline Makefile]: @mr337 - you were correct the other day that to re-inject AWS values I have to specify them explicitly. Ran into this today while trying to write to `chamber` after my session logged out. Fortunately with this change you just exit the container and re-enter and you're set!

File: config/env/experimental.migrations.env [hunk 2]
```diff
@@ -1,5 +1,5 @@
 AWS_S3_BUCKET_NAME=transcom-ppp-orders-experimental-us-west-2
-DB_IAM=false
+DB_IAM=true
```
Comment (chrisgilmerproj) [inline config/env/experimental.migrations.env]: Turning on RDS IAM auth for Experimental.

File: config/env/experimental.orders.env [hunk 3]
```diff
@@ -8,7 +8,7 @@ DEBUG_LOGGING=true
 DOD_CA_PACKAGE=/config/tls/Certificates_PKCS7_v5.6_DoD.der.p7b
 DOMAIN=experimental.move.mil
 HTTP_ORDERS_SERVER_NAME=tsp.experimental.move.mil
-IWS_RBS_ENABLED=1
+IWS_RBS_ENABLED=0
```
Comment (chrisgilmerproj) [inline config/env/experimental.orders.env]: This disables any calls out to DMDC while I upload fake Navy orders.

File: docker-compose.dev.yml [hunk 4]
```diff
@@ -38,7 +38,9 @@ services:
       - AWS_SECURITY_TOKEN
       - AWS_SESSION_EXPIRATION
       - AWS_SESSION_TOKEN
+      - CHAMBER_KMS_KEY_ALIAS=alias/aws/ssm
       - CHAMBER_RETRIES=20
+      - CHAMBER_USE_PATHS=1
```
Comment (chrisgilmerproj) [inline docker-compose.dev.yml]: These changes basically allow me to write secrets to Chamber from inside docker.

**Comment by chrisgilmerproj** [comment]
The proof is in the pudding! https://circleci.com/gh/transcom/milmove_orders/1477

**Review by mr337 — APPROVED** [review]
Happy to see CircleCI run a migration this works! 🚢

**Comment by chrisgilmerproj** [comment]
After getting the DB running I've asked for the count of total orders using this command:

```
 orders-api-client --cac --hostname tsp.experimental.move.mil --port 443 get-orders-count --issuer navy
CAC PIN: ******

{"count":0,"endDateTime":"2020-03-17T14:00:00.004-07:00","issuer":"navy","startDateTime":"2020-03-16T14:00:00.004-07:00"}
```

**Comment by chrisgilmerproj** [comment]
Uploaded fake data!

```
orders-api-client --cac --hostname tsp.experimental.move.mil --port 443 post-revisions --issuer navy --csv-file nom_demo_20190404.csv
CAC PIN: ******

{"edipi":"1234567890","issuer":"navy","ordersNum":"N0002218CTW61XA","revisions":[{"dateIssued":"2017-10-27T00:00:00.000-07:00","gainingUnit":{"city":"ARLINGTON","country":"US","locality":"VA","name":"ARLING","uic":"N64168"},"hasDependents":false,"losingUnit":{"city":"NORFOLK","country":"US","locality":"VA","name":"NORVA","uic":"N21411"},"member":{"affiliation":"navy","familyName":"BELSON","givenName":"Gabriell","rank":"o-3","title":"Lieutenant"},"ntsAccounting":{},"ordersType":"operational","pcsAccounting":{"tac":"NM48"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-12-31","seqNum":2,"status":"authorized","tourType":"accompanied","ubAccounting":{}}],"uuid":"38fdec03-e22f-4095-83ea-d8c86648d536"}
{"edipi":"1234567890","issuer":"navy","ordersNum":"N0002218CTW61U8","revisions":[{"dateIssued":"2017-10-04T00:00:00.000-07:00","gainingUnit":{"city":"NORTH ISLAND","country":"US","locality":"CA","name":"NORTI","uic":"N55139"},"hasDependents":false,"losingUnit":{"city":"MILTON","country":"US","locality":"FL","name":"MILTON","uic":"N44988"},"member":{"affiliation":"navy","familyName":"Verderame","givenName":"Jamaad","rank":"o-2","title":"Lieutenant Junior Grade"},"ntsAccounting":{},"ordersType":"training","pcsAccounting":{"tac":"NM38"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-11-30","seqNum":1,"status":"authorized","tourType":"accompanied","ubAccounting":{}}],"uuid":"fb3b3da6-f746-4492-ad93-3a02ed22bcac"}
{"edipi":"1234567890","issuer":"navy","ordersNum":"N0002218CTW61ZL","revisions":[{"dateIssued":"2017-11-03T00:00:00.000-07:00","gainingUnit":{"city":"GARDEN CITY","country":"US","locality":"NY","name":"GARDEN","uic":"N62445"},"hasDependents":false,"losingUnit":{"city":"CORONADO","country":"US","locality":"CA","name":"CORNDO","uic":"N46462"},"member":{"affiliation":"navy","familyName":"Tech","givenName":"Saymon","rank":"o-3","title":"Lieutenant"},"ntsAccounting":{},"ordersType":"training","pcsAccounting":{"tac":"NM38"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-11-30","seqNum":3,"status":"authorized","tdyEnRoute":true,"tourType":"accompanied","ubAccounting":{}}],"uuid":"6bfef179-3d5b-4720-86c2-92af1b30789f"}
{"edipi":"1234567890","issuer":"navy","ordersNum":"N0002218CTW620Q","revisions":[{"dateIssued":"2018-06-12T00:00:00.000-07:00","gainingUnit":{"city":"OAK HARBOR","country":"US","locality":"WA","name":"O HARB","uic":"N4353A"},"hasDependents":true,"losingUnit":{"city":"STATE COLLEGE","country":"US","locality":"PA","name":"STATEC","uic":"N63310"},"member":{"affiliation":"navy","familyName":"Weatherford","givenName":"Turley","rank":"o-3","title":"Lieutenant"},"ntsAccounting":{},"ordersType":"operational","pcsAccounting":{"tac":"NM48"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2018-10-31","seqNum":1,"status":"authorized","tdyEnRoute":true,"tourType":"accompanied","ubAccounting":{}}],"uuid":"f7f04c02-a3f2-419c-8ea6-f93136218e33"}
{"edipi":"1234567890","issuer":"navy","ordersNum":"N0002218CTW61PN","revisions":[{"dateIssued":"2017-10-11T00:00:00.000-07:00","gainingUnit":{"city":"NORTH ISLAND","country":"US","locality":"CA","name":"NORTI","uic":"N55139"},"hasDependents":false,"losingUnit":{"city":"MILTON","country":"US","locality":"FL","name":"MILTON","uic":"N44988"},"member":{"affiliation":"navy","familyName":"KIRSTENRIVAS","givenName":"Liette","middleName":"Victoria","rank":"o-2","title":"Lieutenant Junior Grade"},"ntsAccounting":{},"ordersType":"training","pcsAccounting":{"tac":"NM38"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-11-30","seqNum":2,"status":"authorized","tourType":"accompanied","ubAccounting":{}}],"uuid":"bb72dcb7-91c9-4b00-9d4b-c9198824304d"}
{"edipi":"1234567890","issuer":"navy","ordersNum":"N0002218CTW620R","revisions":[{"dateIssued":"2017-10-04T00:00:00.000-07:00","gainingUnit":{"city":"SAN DIEGO","country":"US","locality":"CA","name":"SDGO","uic":"N21465"},"hasDependents":false,"losingUnit":{"city":"CHARLESTON","country":"US","locality":"SC","name":"CHARLE","uic":"N47801"},"member":{"affiliation":"navy","familyName":"Harris-Grant","givenName":"Johnie","rank":"o-2","title":"Lieutenant Junior Grade"},"ntsAccounting":{},"ordersType":"training","pcsAccounting":{"tac":"NM38"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-11-30","seqNum":1,"status":"authorized","tourType":"accompanied","ubAccounting":{}}],"uuid":"bb82d16d-c4bf-4afc-8348-6109f4b88199"}
{"edipi":"1234567890","issuer":"navy","ordersNum":"N0002218CTW6249","revisions":[{"dateIssued":"2018-05-21T00:00:00.000-07:00","gainingUnit":{"city":"SAN DIEGO","country":"JA","locality":"CA","name":"SDGO","uic":"N22202"},"hasDependents":false,"losingUnit":{"city":"SAN DIEGO","country":"US","locality":"CA","name":"SDGO","uic":"N55687"},"member":{"affiliation":"navy","familyName":"Seamaster","givenName":"Azantae","rank":"o-2","title":"Lieutenant Junior Grade"},"ntsAccounting":{},"ordersType":"operational","pcsAccounting":{"tac":"NM48"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2018-10-31","seqNum":4,"status":"authorized","tdyEnRoute":true,"tourType":"accompanied","ubAccounting":{}}],"uuid":"14a0d4e8-d99f-4041-96bc-61124337d892"}
```

**Comment by chrisgilmerproj** [comment]
Orders are on experimental! 

```
orders-api-client --cac --hostname tsp.experimental.move.mil --port 443 get-orders-count --issuer navy --relative-time -1m
CAC PIN: ******

{"count":7,"issuer":"navy"}
```

**Comment by chrisgilmerproj** [comment]
Was able to fix the relative-time search by using UTC time:

```
orders-api-client --cac --hostname tsp.experimental.move.mil --port 443 get-orders-count --issuer navy
CAC PIN: ******

{"count":7,"endDateTime":"2020-03-17T21:10:31.669Z","issuer":"navy","startDateTime":"2020-03-16T21:10:31.669Z"}
```

**Comment by chrisgilmerproj** [comment]
As a follow up I uploaded E-Orders Revisions to the same orders as before today around 12:25 PST. The following is more FAKE data.

```
orders-api-client --cac --hostname tsp.experimental.move.mil --port 443 post-revisions --issuer navy --csv-file nom_demo_20190404_rev_update.csv
CAC PIN: ******

{"edipi":"1234567890","issuer":"navy","ordersNum":"N0002218CTW61XA","revisions":[{"dateIssued":"2017-10-27T00:00:00.000Z","gainingUnit":{"city":"ARLINGTON","country":"US","locality":"VA","name":"ARLING","uic":"N64168"},"hasDependents":false,"losingUnit":{"city":"NORFOLK","country":"US","locality":"VA","name":"NORVA","uic":"N21411"},"member":{"affiliation":"navy","familyName":"BELSON","givenName":"Gabriell","rank":"o-3","title":"Lieutenant"},"ntsAccounting":{},"ordersType":"operational","pcsAccounting":{"tac":"NM48"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-12-31","seqNum":2,"status":"authorized","tourType":"accompanied","ubAccounting":{}},{"dateIssued":"2017-10-27T00:00:00.000-07:00","gainingUnit":{"city":"ARLINGTON","country":"US","locality":"VA","name":"ARLING","uic":"N64168"},"hasDependents":false,"losingUnit":{"city":"NORFOLK","country":"US","locality":"VA","name":"NORVA","uic":"N21411"},"member":{"affiliation":"navy","familyName":"BELSON","givenName":"Gabriell","rank":"o-3","title":"Lieutenant"},"ntsAccounting":{},"ordersType":"operational","pcsAccounting":{"tac":"NM48"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-12-31","seqNum":3,"status":"authorized","tourType":"accompanied","ubAccounting":{}}],"uuid":"38fdec03-e22f-4095-83ea-d8c86648d536"}
{"edipi":"1234567890","issuer":"navy","ordersNum":"N0002218CTW61U8","revisions":[{"dateIssued":"2017-10-04T00:00:00.000Z","gainingUnit":{"city":"NORTH ISLAND","country":"US","locality":"CA","name":"NORTI","uic":"N55139"},"hasDependents":false,"losingUnit":{"city":"MILTON","country":"US","locality":"FL","name":"MILTON","uic":"N44988"},"member":{"affiliation":"navy","familyName":"Verderame","givenName":"Jamaad","rank":"o-2","title":"Lieutenant Junior Grade"},"ntsAccounting":{},"ordersType":"training","pcsAccounting":{"tac":"NM38"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-11-30","seqNum":1,"status":"authorized","tourType":"accompanied","ubAccounting":{}},{"dateIssued":"2017-10-04T00:00:00.000-07:00","gainingUnit":{"city":"NORTH ISLAND","country":"US","locality":"CA","name":"NORTI","uic":"N55139"},"hasDependents":false,"losingUnit":{"city":"MILTON","country":"US","locality":"FL","name":"MILTON","uic":"N44988"},"member":{"affiliation":"navy","familyName":"Verderame","givenName":"Jamaad","rank":"o-2","title":"Lieutenant Junior Grade"},"ntsAccounting":{},"ordersType":"training","pcsAccounting":{"tac":"NM38"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-11-30","seqNum":2,"status":"authorized","tourType":"accompanied","ubAccounting":{}}],"uuid":"fb3b3da6-f746-4492-ad93-3a02ed22bcac"}
{"edipi":"1234567890","issuer":"navy","ordersNum":"N0002218CTW61ZL","revisions":[{"dateIssued":"2017-11-03T00:00:00.000Z","gainingUnit":{"city":"GARDEN CITY","country":"US","locality":"NY","name":"GARDEN","uic":"N62445"},"hasDependents":false,"losingUnit":{"city":"CORONADO","country":"US","locality":"CA","name":"CORNDO","uic":"N46462"},"member":{"affiliation":"navy","familyName":"Tech","givenName":"Saymon","rank":"o-3","title":"Lieutenant"},"ntsAccounting":{},"ordersType":"training","pcsAccounting":{"tac":"NM38"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-11-30","seqNum":3,"status":"authorized","tdyEnRoute":true,"tourType":"accompanied","ubAccounting":{}},{"dateIssued":"2017-11-03T00:00:00.000-07:00","gainingUnit":{"city":"GARDEN CITY","country":"US","locality":"NY","name":"GARDEN","uic":"N62445"},"hasDependents":false,"losingUnit":{"city":"CORONADO","country":"US","locality":"CA","name":"CORNDO","uic":"N46462"},"member":{"affiliation":"navy","familyName":"Tech","givenName":"Saymon","rank":"o-3","title":"Lieutenant"},"ntsAccounting":{},"ordersType":"training","pcsAccounting":{"tac":"NM38"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-11-30","seqNum":4,"status":"authorized","tdyEnRoute":true,"tourType":"accompanied","ubAccounting":{}}],"uuid":"6bfef179-3d5b-4720-86c2-92af1b30789f"}
{"edipi":"1234567890","issuer":"navy","ordersNum":"N0002218CTW620Q","revisions":[{"dateIssued":"2018-06-12T00:00:00.000Z","gainingUnit":{"city":"OAK HARBOR","country":"US","locality":"WA","name":"O HARB","uic":"N4353A"},"hasDependents":true,"losingUnit":{"city":"STATE COLLEGE","country":"US","locality":"PA","name":"STATEC","uic":"N63310"},"member":{"affiliation":"navy","familyName":"Weatherford","givenName":"Turley","rank":"o-3","title":"Lieutenant"},"ntsAccounting":{},"ordersType":"operational","pcsAccounting":{"tac":"NM48"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2018-10-31","seqNum":1,"status":"authorized","tdyEnRoute":true,"tourType":"accompanied","ubAccounting":{}},{"dateIssued":"2018-06-12T00:00:00.000-07:00","gainingUnit":{"city":"OAK HARBOR","country":"US","locality":"WA","name":"O HARB","uic":"N4353A"},"hasDependents":true,"losingUnit":{"city":"STATE COLLEGE","country":"US","locality":"PA","name":"STATEC","uic":"N63310"},"member":{"affiliation":"navy","familyName":"Weatherford","givenName":"Turley","rank":"o-3","title":"Lieutenant"},"ntsAccounting":{},"ordersType":"operational","pcsAccounting":{"tac":"NM48"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2018-10-31","seqNum":2,"status":"authorized","tdyEnRoute":true,"tourType":"accompanied","ubAccounting":{}}],"uuid":"f7f04c02-a3f2-419c-8ea6-f93136218e33"}
{"edipi":"1234567890","issuer":"navy","ordersNum":"N0002218CTW61PN","revisions":[{"dateIssued":"2017-10-11T00:00:00.000Z","gainingUnit":{"city":"NORTH ISLAND","country":"US","locality":"CA","name":"NORTI","uic":"N55139"},"hasDependents":false,"losingUnit":{"city":"MILTON","country":"US","locality":"FL","name":"MILTON","uic":"N44988"},"member":{"affiliation":"navy","familyName":"KIRSTENRIVAS","givenName":"Liette","middleName":"Victoria","rank":"o-2","title":"Lieutenant Junior Grade"},"ntsAccounting":{},"ordersType":"training","pcsAccounting":{"tac":"NM38"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-11-30","seqNum":2,"status":"authorized","tourType":"accompanied","ubAccounting":{}},{"dateIssued":"2017-10-11T00:00:00.000-07:00","gainingUnit":{"city":"NORTH ISLAND","country":"US","locality":"CA","name":"NORTI","uic":"N55139"},"hasDependents":false,"losingUnit":{"city":"MILTON","country":"US","locality":"FL","name":"MILTON","uic":"N44988"},"member":{"affiliation":"navy","familyName":"KIRSTENRIVAS","givenName":"Liette","middleName":"Victoria","rank":"o-2","title":"Lieutenant Junior Grade"},"ntsAccounting":{},"ordersType":"training","pcsAccounting":{"tac":"NM38"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-11-30","seqNum":3,"status":"authorized","tourType":"accompanied","ubAccounting":{}}],"uuid":"bb72dcb7-91c9-4b00-9d4b-c9198824304d"}
{"edipi":"1234567890","issuer":"navy","ordersNum":"N0002218CTW620R","revisions":[{"dateIssued":"2017-10-04T00:00:00.000Z","gainingUnit":{"city":"SAN DIEGO","country":"US","locality":"CA","name":"SDGO","uic":"N21465"},"hasDependents":false,"losingUnit":{"city":"CHARLESTON","country":"US","locality":"SC","name":"CHARLE","uic":"N47801"},"member":{"affiliation":"navy","familyName":"Harris-Grant","givenName":"Johnie","rank":"o-2","title":"Lieutenant Junior Grade"},"ntsAccounting":{},"ordersType":"training","pcsAccounting":{"tac":"NM38"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-11-30","seqNum":1,"status":"authorized","tourType":"accompanied","ubAccounting":{}},{"dateIssued":"2017-10-04T00:00:00.000-07:00","gainingUnit":{"city":"SAN DIEGO","country":"US","locality":"CA","name":"SDGO","uic":"N21465"},"hasDependents":false,"losingUnit":{"city":"CHARLESTON","country":"US","locality":"SC","name":"CHARLE","uic":"N47801"},"member":{"affiliation":"navy","familyName":"Harris-Grant","givenName":"Johnie","rank":"o-2","title":"Lieutenant Junior Grade"},"ntsAccounting":{},"ordersType":"training","pcsAccounting":{"tac":"NM38"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2017-11-30","seqNum":2,"status":"authorized","tourType":"accompanied","ubAccounting":{}}],"uuid":"bb82d16d-c4bf-4afc-8348-6109f4b88199"}
{"edipi":"1234567890","issuer":"navy","ordersNum":"N0002218CTW6249","revisions":[{"dateIssued":"2018-05-21T00:00:00.000Z","gainingUnit":{"city":"SAN DIEGO","country":"JA","locality":"CA","name":"SDGO","uic":"N22202"},"hasDependents":false,"losingUnit":{"city":"SAN DIEGO","country":"US","locality":"CA","name":"SDGO","uic":"N55687"},"member":{"affiliation":"navy","familyName":"Seamaster","givenName":"Azantae","rank":"o-2","title":"Lieutenant Junior Grade"},"ntsAccounting":{},"ordersType":"operational","pcsAccounting":{"tac":"NM48"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2018-10-31","seqNum":4,"status":"authorized","tdyEnRoute":true,"tourType":"accompanied","ubAccounting":{}},{"dateIssued":"2018-05-21T00:00:00.000-07:00","gainingUnit":{"city":"SAN DIEGO","country":"JA","locality":"CA","name":"SDGO","uic":"N22202"},"hasDependents":false,"losingUnit":{"city":"SAN DIEGO","country":"US","locality":"CA","name":"SDGO","uic":"N55687"},"member":{"affiliation":"navy","familyName":"Seamaster","givenName":"Azantae","rank":"o-2","title":"Lieutenant Junior Grade"},"ntsAccounting":{},"ordersType":"operational","pcsAccounting":{"tac":"NM48"},"povShipmentAccounting":{},"povStorageAccounting":{},"reportNoLaterThan":"2018-10-31","seqNum":5,"status":"authorized","tdyEnRoute":true,"tourType":"accompanied","ubAccounting":{}}],"uuid":"14a0d4e8-d99f-4041-96bc-61124337d892"}
```

As expected, the number of orders did not change since we only added revisions:

```
orders-api-client --cac --hostname tsp.experimental.move.mil --port 443 get-orders-count --issuer navy --relative-time 72h
CAC PIN: ******

{"count":7,"endDateTime":"2020-03-19T19:26:57.179Z","issuer":"navy","startDateTime":"2020-03-16T19:26:57.179Z"}
```

**Pull request merged**